pkgs/development/python-modules/mohawk/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, python, mock, nose, pytest, six }:
```

personal preference to separate inputs from expression
```suggestion
{ lib, buildPythonPackage, fetchPypi, python, mock, nose, pytest, six }:

```

---

Done!